### PR TITLE
Small edits to Ranking Evaluation API docs

### PR DESCRIPTION
**Pull request by liebeslied**
For the sake of clarity, fix some small grammar/syntax issues and change some wordings.

### CODE REVIEW

File: docs/reference/search/rank-eval.asciidoc [hunk 1]
```diff
@@ -346,16 +346,16 @@ http://olivier.chapelle.cc/pub/err.pdf[Expected reciprocal rank for graded relev
...
 information needs where the user is interested in finding many relevant 
 documents in the top k results.
 
 The metric models the expectation of the reciprocal of the position at which a 
-user stops reading through the result list. This means that relevant document in 
-top ranking positions will contribute much to the overall score. However, the 
-same document will contribute much less to the score if it appears in a lower 
-rank, even more so if there are some relevant (but maybe less relevant) 
-documents preceding it. In this way, the ERR metric discounts documents which 
+user stops reading through the result list. This means that a relevant documents
```
Comment (cbuescher) [inline docs/reference/search/rank-eval.asciidoc]:
```suggestion
user stops reading through the result list. This means that a relevant document
```
I think this should be singular now after your changes.

**Review by cbuescher — APPROVED** [review]
Hi @liebeslied, thanks for opening the PR, it great to see how much this improves the readability of this particular section in the docs. Many thanks for that. I left a small request for change (if you agree with me), other than that I will run some requires CI tests and will merge as soon as you were able to reply to my suggestion below (... and I will work on my use of the possessive `'s` in English, as a non-native speaker, having written most of these docs its great to get this kind of feedback).

File: docs/reference/search/rank-eval.asciidoc (hunk 1 above)
Comment (liebeslied) [reply docs/reference/search/rank-eval.asciidoc]: nice catch, thanks!
I'm glad it's helpful--I love copy editing :)

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Comment by cbuescher** [comment]
Thanks again @liebeslied, merged the PR to 7.5 and newer branches.

**Pull request merged**